diff --git a/node_modules/node-notifier-cli/bin.js b/node_modules/node-notifier-cli/bin.js
index 9385f22..da68d05 100755
--- a/node_modules/node-notifier-cli/bin.js
+++ b/node_modules/node-notifier-cli/bin.js
@@ -74,6 +74,7 @@ function doNotification(options) {
     // Do not show an empty message
     process.exit(0);
   }
+  options = Object.assign({}, options, { timeout: false });
   notifier.notify(options, function(err, msg) {
     if (err) {
       console.error(err.message);
